web/README.md

```diff
@@ -20,14 +20,17 @@ sudo apt-get install nodejs
 
 The following folders contain the distribution for KeymanWeb:
 
-	source				         Source code
-	intermediate    	         Intermediate compiled code; TypeScript source is compiled here before it is minified.
-	release/web	                 Fully-compiled KeymanWeb modules
-	release/embedded	         Fully-compiled KMEA/KMEI modules for inclusion in mobile app builds
+	source				Source code
+	unit_tests      	Automated testing resources
+  
+	intermediate    	Intermediate compiled code; TypeScript source is compiled here before it is minified.
+	release/web			Fully-compiled KeymanWeb modules for testing
+	release/embedded	Fully-compiled KMEA/KMEI modules for inclusion in mobile app builds
 	release/unminified/web       Fully-compiled but non-minified KeymanWeb modules
 	release/unminified/embedded  Fully-compiled KMEA/KMEI modules for inclusion in mobile app builds
-	samples				         Sample pages demonstrating ways to link with KeymanWeb
-	testing     		         Test-case web-pages for various aspects of KeymanWeb functionality
+  
+	samples				Sample pages demonstrating ways to link with KeymanWeb
+	testing     		Test-case web-pages for various aspects of KeymanWeb functionality
 
 ## Usage
 Open **index.html** or **samples/index.html** in your browser. Be sure to compile KeymanWeb before viewing the pages.
```

Comment:
It would be helpful to add a short section
`### Unit Test`

to tell developers how to run the tests. (prefer `npm test` or `unit_tests/test.sh`?)
Maybe include the test.sh options too.